unitTests/utils/FacetUtilsTest.ts

```diff
@@ -33,5 +33,25 @@ export const FacetUtilsTest = () => {
         expect(translation).toEqual(month.translation);
       });
     });
+
+    describe('testing getDisplayValueFromValueCaption', () => {
+      it(`when there is no value caption
+        display value should be equal to the value`, () => {
+        const displayValue = FacetUtils.getDisplayValueFromValueCaption('allo', '@random', {})
+        expect(displayValue).toBe('allo');
+      });
+
+      it(`when there is a value caption
+      display value should be equal to the caption for that value`, () => {
+        const displayValue = FacetUtils.getDisplayValueFromValueCaption('allo', '@random', { allo: 'bye' })
+        expect(displayValue).toBe('bye');
+      });
+
+      it('should call tryToGetTranslatedCaption from FacetUtils', () => {
```

Comment:
Hmm, I don't think this test is needed because it asserts a specific internal implementation.
I think just asserting that the value caption is resolved correctly in the test on line 44 is enough, while keeping the internal implementation flexible (e.g. to rename)